### PR TITLE
St convert to hooks

### PR DESCRIPTION
**Pull request by SamaadTurner**

### CODE REVIEW

File: src/User/Profile.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { useAuth0 } from "@auth0/auth0-react";
-import ProfileView from "./ProvileView";
+import ProfileView from "./ProfileView"
```
Comment (kamsci) [inline src/User/Profile.jsx:2]:
```suggestion
import ProfileView from "./ProfileView";
```

File: src/HelperComponents/PaginationComponent.jsx [hunk 2]
```diff
@@ -1,94 +1,81 @@
...
     }
-
-    renderAdvancedPagination(lastPage) {
-        const middlePage = this.calculateMidPage(lastPage);
+    const renderAdvancedPagination = (lastPage) => {
+        const middlePage = calculateMidPage(lastPage);
         const preMiddlePage = middlePage - 1;
         const postMiddlePage = middlePage + 1;
-        const { activePage } = this.state;
+        activePage;
```
Comment (kamsci) [inline src/HelperComponents/PaginationComponent.jsx:33]:
```suggestion
```

random reference of activePage.. it doesn't do anything here, you already have activePage declared in the function. you can remove this line.

**Review by kamsci — CHANGES_REQUESTED** [review]
I don't see any components from the src/Gardens/SearchView folder changed. 

Can you go through and change those as well?

**Review by SamaadTurner — COMMENTED** [review]
got it.

**Review by kamsci — APPROVED** [review]

**Pull request merged**